content/dotnet-template-azure-iot-edge-module/CSharp/ModuleBackgroundService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Azure.Devices.Client;
```

Comment:
Should these files have a licensing header?

e.g.:
> // Copyright (c) Microsoft. All rights reserved.
> // Licensed under the MIT license. See LICENSE file in the project root for full license information.

Reply:
I've not seen license headers on templates, they are intended to provide a starting point to the user, and they will make changes.